.devcontainer/images/.claude/commands/do/decompose.md

```
@@ -0,0 +1,45 @@
+
+  2_prioritize:
+    action: "Order by dependency"
+    principle: "Smallest change first"
+
+  3_create_todos:
+    action: "Initialize TaskCreate with sub-objectives"
+```
+
+**Output Phase 5.0:**
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)